web-ui/src/components/reviews/periods/ReviewPeriods.jsx

```diff
@@ -457,17 +473,12 @@ const ReviewPeriods = ({ onPeriodSelected, mode }) => {
                     key={`period-lia-${id}`}
                     onClick={() => onPeriodClick(id)}
                   >
-                    <Avatar>
-                      <WorkIcon />
-                    </Avatar>
+                    <Avatar>{reviewStatusIconMap[reviewStatus]}</Avatar>
                   </ListItemAvatar>
                   <ListItemText
                     key={`period-lit-${id}`}
                     onClick={() => onPeriodClick(id)}
-                    primary={
-                      name +
-                      (reviewStatus === ReviewStatus.OPEN ? ' - Open' : '')
-                    }
+                    primary={`${name} - ${titleCase(reviewStatus)}`}
```

Comment:
We now display the actual status string instead of only displaying "Open" or nothing.